connectors/class-connector-woocommerce.php

```diff
@@ -488,19 +505,21 @@ public function callback_woocommerce_order_status_changed( $order_id, $old, $new
 			$old_status_name = $old_status->name;
 		}
 
-		/* translators: %1$s: an order title, %2$s: order status, %3$s: another order status (e.g. "Order #42", "processing", "complete") */
-		$message = esc_html_x(
-			'%1$s status changed from %2$s to %3$s',
-			'1. Order title, 2. Old status, 3. New status',
-			'stream'
+		$order       = new \WC_Order( $order_id );
```

Comment:
I'm fairly certain current convention is to use getters, in this `wc_get_order()`